.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Install toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.70.0
+          toolchain: 1.79.0
```

Comment:
This one is pretty fresh, it would be nice if `rustc-perf` compiled at least with a few older versions of the compiler.

Reply:
To which version should i drop?

Reply:
Ideally the oldest one that still compiles :) Looks like `1.73.0` should be enough.

Reply:
Well, locally i can build as old as 1.78, because 1.77 will give
`
link: extra operand 'rustc-perf\\target\\debug\\build\\windows_x86_64_msvc-f220d21a5bc470a5\\build_script_build-f220d21a5bc470a5.build_script_build.d44e75c843d0cc87-cgu.0.rcgu.o'
`

Reply:
Bumped to 1.73